CODE-OF-CONDUCT.md

```diff
@@ -56,7 +56,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at oss-coc@vmware.com. All
+reported by contacting the project team at oss-coc@omnissa.com. All
```

Comment:
I'm not sure if we have `oss-coc@omnissa.com`, but for now, let's just assume we do have it